### PR TITLE
PAYEE-746 - Removing exception

### PR DESCRIPTION
**Pull request by brendanstraughan**
- [ ] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

File: app/controllers/IncomeController.scala [hunk 1]
```diff
@@ -214,7 +214,8 @@ class IncomeController @Inject()(
 
                     case _ => throw new RuntimeException(s"Not able to found employment with id $empId")
                   }
-                case _ => throw new RuntimeException("Exception while reading employment and tax code details")
+                case _ =>
+                  errorPagesHandler.internalServerError("Exception while reading employment and tax code details")
```
Comment (sergi-hmrc) [inline app/controllers/IncomeController.scala:218]: should we log any of the errors that may have occurred?

**Review by sergi-hmrc — APPROVED** [review]

File: app/controllers/IncomeController.scala (hunk 1 above)
Comment (PerrettJ4) [reply app/controllers/IncomeController.scala:218]: I'm confused, so as a user where will I be directed?

Comment (brendanstraughan) [reply app/controllers/IncomeController.scala:218]: they will just go to the standard internal server error page @PerrettJ4

Comment (brendanstraughan) [reply app/controllers/IncomeController.scala:218]: @sergi-hmrc would we do this in a recover block or like how you have done yours?

Comment (sergi-hmrc) [reply app/controllers/IncomeController.scala:218]: my bad this method logs... sorry

**Review by masahmad — APPROVED** [review]

**Review by sergi-hmrc — APPROVED** [review]

**Pull request merged**